clouddriver-docker/src/test/groovy/com/netflix/spinnaker/clouddriver/docker/registry/config/DockerRegistryConfigurationPropertiesSpec.groovy

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.docker.registry.config
+
+import spock.lang.Specification
+
+class DockerRegistryConfigurationPropertiesSpec extends Specification {
```

Comment:
Ideally new code (even test code) would not be in groovy.  Any chance you can convert this to java?  Thanks!